.github/workflows/publish.yml

```
@@ -0,0 +1,42 @@
+        run: pnpm build
+        working-directory: packages/ui-base
+
+      - name: Run lint
+        run: pnpm lint
+        working-directory: packages/ui-base
+
+      - name: Publish to NPM
+        run: pnpm publish --no-git-checks
+        working-directory: packages/ui-base
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)